### ISSUE TITLE
Duplicate messages sent

### ISSUE THREAD

**Issue by rugk** [threema-ch/threema-web#660]
## Expected Behavior

Message sent only once.

## Current Behavior

Sends message twice.

## Steps to Reproduce (for bugs)

I have one session on one device, where this always happens.
I cannot reproduce it on another device.

## Your Environment

* Threema Web version: 2.1.1-gh
* Browser name and version: Firefox 63
* Computer operating system and version: Fedora 29

**Comment by dbrgn** [comment]
Seems to be a duplicate of #481, so I'll close this and reopen the other issue.

Can you send us the message log of the session where you can reproduce this? Please refer to this issue, so that support staff will assign the ticket to me.

Note: I'm pretty sure that it will not be reproducible anymore after you kill the app or deactivate Threema Web.
